flawless/server/service.py

```diff
@@ -239,6 +239,8 @@ def _format_traceback(self, err_info, append_locals=True, show_full_stack=False,
                 b=start_bold, xb=end_bold, count=err_info.error_count, lb=linebreak))
             parts.append("{b}Last Occurred:{xb} {last_occurrence}{lb}".format(
                 b=start_bold, xb=end_bold, last_occurrence=err_info.last_occurrence, lb=linebreak))
+            parts.append("{b}Hostname:{xb} {hostname}{lb}".format(
```

Comment:
Could you change `{b}Hostname:{xb}` to `{b}Last Occurrence Logged From:{xb}` ?

Other than that, lgtm